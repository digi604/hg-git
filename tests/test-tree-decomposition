#!/bin/sh

"$TESTDIR/hghave" git || exit 80

echo "[extensions]" >> $HGRCPATH
echo "bookmarks = " >> $HGRCPATH

hg init
git init -q

mkdir d1
touch d1/f1 d1/f2
git add d1/f1 d1/f2
git commit -qm initial

mkdir d2
git mv d1/f2 d2/f2
git commit -qm rename

hg gimport
hg log -r tip --template 'adds: {file_adds}\ndels: {file_dels}\n'
