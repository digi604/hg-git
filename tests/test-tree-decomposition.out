importing Git objects into Hg
at: 0/2
adds: d2/f2
dels: d1/f2
